docs/integrations/containers-orchestration/vmware.md

```diff
@@ -153,9 +153,9 @@ python3 events.py  -s 192.168.23.242 -t vcenterhost -to 1514 -bT '2012-10-08 00:
 * **-o**: Remote vCenter Server port to use, default 443. Optional.
 * **-u**: User name to use when connecting to vCenter server. Required.
 * **-p**: Password to use when connecting to vCenter server. Required.
-* **-f**: Output File Prefix. Target syslog server or file is required.
+* **-f**: Output File Prefix. Target syslog server or file is required. Optional.
 * **-ts**: Timestamp File. Default ‘.timelog_events’. Optional.
-* **-t**: Target Sumologic syslog server. Target syslog server or file is required.
+* **-t**: Sumologic installed collector host name. This is required in case of syslog server. If using local file source this parameter is not required.
```

Comment:
```suggestion
* **-t**: Host name of Sumo Logic installed collector. This is required in case of syslog server. If you are using local file source this parameter is not required.
```